integration-tests/src/test/java/org/zowe/apiml/integration/zaas/ZaasNegativeTest.java

```diff
@@ -51,6 +51,7 @@ public class ZaasNegativeTest {
 
     private static final Set<URI> endpoints = new HashSet<URI>() {{
         add(ZAAS_TICKET_URI);
+        add(ZAAS_SAFIDT_URI);
```

Comment:
It's a little bit of a pity to miss`givenClientAndHeaderCertificates_thenReturnTokenFromClientCert` because it makes perfect sense to test that the correct certificate was used. But I don't know what is the best way to use it when the request is slightly different.

Reply:
It is possible to create a stream for parallel tests with correct requests. Ie. Producer of the request as an argument of test method. The second possibility is to make this class abstract and implement twice with a different construction/verification of request.

Reply:
I've added the rest assured RequestSpecification (given() part) as part of Arguments for the Parameterized tests. This way the requests are correctly constructed for each uri.